games/minetest/Portfile

```diff
@@ -44,27 +45,44 @@ post-extract {
     }
 }
 
-license                 LGPL-2.1+
+license                 LGPL
 categories              games
 platforms               darwin
-maintainers             nomaintainer
+maintainers             @Zweihorn
 description             open source infinite-world block sandbox game with survival and crafting
-long_description        ${description}.
-
-depends_build-append    path:bin/doxygen:doxygen \
-                        port:mesa
+long_description        - ${description} - \
```

Comment:
this long piece of texts is is IMHO not really needed - does this add any useful information for the end-user?

Reply:
OK - **Too much is too much** and the `port info minetest` does not meet my expectations due to garbled text.

Shortened to:
```
long_description        ${description} - \
                        Find the MT mods at <https://content.minetest.net/> and have fun.
```

Adding the MT mods will greatly expand the user experience of the MT game. ContenDB is the main source for MT mods as preferred by the MT project team. However, many MT players seem to remain unaware of this although advertised on the MT homepage. This the info.

My new _text tsunami_ was discarded for the good cause.